package.json

```diff
@@ -52,7 +52,7 @@
     "lint-staged": "lint-staged",
     "next-with-deps": "./scripts/next-with-deps.sh",
     "next": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation --enable-source-maps packages/next/dist/bin/next",
-    "next-no-sourcemaps": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation packages/next/dist/bin/next",
```

Comment:
I can't just add `--disable-source-maps` here since our args parser (Commander) would treat it as the positional. Yargs would treat it different e.g. `next --foo bar` would be equivalent to `next bar --foo` but is not with Commander.